MapboxVision.xcodeproj/project.pbxproj

```diff
@@ -98,15 +95,21 @@
 		3A11B044229ED3F500070A61 /* ARRendererError.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A11B043229ED3F500070A61 /* ARRendererError.swift */; };
 		3A11B046229ED49B00070A61 /* UIColor+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A11B045229ED49B00070A61 /* UIColor+Extension.swift */; };
 		3A11B05E229F1E1900070A61 /* ARSceneTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A11B05D229F1E1900070A61 /* ARSceneTests.swift */; };
+		3A11FEDF2282BCD300B68E5B /* DeviceCheckerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A11FEDE2282BCD300B68E5B /* DeviceCheckerTests.swift */; };
 		3A2D99BE22C347A300F15218 /* LaneVisualParams.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A2D99BD22C347A300F15218 /* LaneVisualParams.swift */; };
+		3A55C6AF22E5E6830048F5C0 /* PerformanceProvider.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A55C6AE22E5E6830048F5C0 /* PerformanceProvider.swift */; };
 		3A5E8C0722A5239F00CC0954 /* Uniforms.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A5E8C0622A5239F00CC0954 /* Uniforms.swift */; };
+		3A7676AE22D647FE00ABC483 /* DeviceChecker.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2E19772A211DDD5900311B95 /* DeviceChecker.swift */; };
 		3A7E992722A005A1005EB875 /* (null) in CopyFiles */ = {isa = PBXBuildFile; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
Wrong entry is in the project file.

Reply:
Interesting... Looks like rebase issue. Good catch, thank you @chezzdev !

Reply:
Looks like this's a side-effect of Xcode. I've tried to revert commit with changes in `project.pbxproj` and then I've readded our new files. Result are the same.

Reply:
Seems like this line was added not in this PR. But it may be removed here

Reply:
Yeah, that's strange, but I'm not sure it's a right solution to manually edit `.pbxproj` file.

These lines are being added during adding BatteryManager/BluetoothManager/PerformanceProvider files. Probably Xcode is restructuring project somehow - deletes DeviceChecker.swift from one place and add DeviceChecker.swift to another place.

You can try to revert `project.pbxproj` file and add files manually. I've did it, but for some reason the behavior remains the same as above.

I'll try analyze it later today.

Reply:
I'm talking just about `(null) in CopyFiles` line. Diff doesn't show that it's added in this PR. I'm sure it's safe to remove it manually.

Reply:
Oh, I'm sorry for misunderstanding, I thought you're talking about `DeviceChecker` and so on.
Sure, will do, thanks!